deploy-board/deploy_board/templates/accounts/accounts.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'accounts/accounts_base.html' %}
+
+{% block breadcrumb-items %}
+<ul class="breadcrumb">
+    <li><a href="/">Home</a></li>
+    <li class="active">Accounts</li>
+</ul>
+{% endblock %}
+
+{% block main %}
+<div class="panel panel-default">
+    <div class="panel-heading clearfix">
+        <h4 class="panel-title pull-left">Account Details</h4>
+    </div>
+    <div class="panel-body table-responsive">
+        <table class="table table-striped table-bordered table-condensed table-hover">
+            <thead>
+                <tr>
+                    <th>ID</th>
+                    <th class="col-lg-2">Name</th>
+                    <th class="col-lg-2">Cell</th>
+                    <th class="col-lg-2">Provider</th>
+                    <th class="col-lg-2"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for account in accounts %}
+                <tr>
+                    <td>{{ account.id }}</td>
```

Comment:
Display `ownerId` from the field `data.ownerId` on the top level for an account record with the `id` field.
`ownerId` contains AWS Account number and it's a primary interest for our users.

`id` is an unique identifier inside our system, but our users have minor interest for this field.

Reply:
do we want to showthe account.id at all, or should we hide it and have owner id be the id. I'm not sure if it would be confusing to a user unfamiliar with how aws account ids look like.

Reply:
added a tooltip so that this shouldn't be an issue.
<img width="240" alt="Screenshot 2024-03-13 at 9 43 16 AM" src="https://github.com/pinterest/teletraan/assets/104773032/0fb5ba8d-bfac-428d-9405-e4f6f048bea7">

Reply:
Let's show account.id as well for internal communication